src/account/UpgradeableModularAccount.sol

```diff
@@ -536,4 +570,20 @@ contract UpgradeableModularAccount is
 
     // solhint-disable-next-line no-empty-blocks
     function _authorizeUpgrade(address newImplementation) internal override {}
+
+    function _checkPermittedCallerIfNotFromEP() internal view {
+        AccountStorage storage _storage = getAccountStorage();
+
+        if (_storage.selectorData[msg.sig].denyExecutionCount > 0) {
+            revert AlwaysDenyRule();
```

Comment:
Kinda want to revisit naming of AlwaysDenyRule and denyExecutionCount too.

Reply:
Or we can get rid of the magic values in #63, if that is finalized